apps/molgenis-components/src/client/client.js

```diff
@@ -154,11 +155,58 @@ const request = async (url, graqphl) => {
   return result.data.data;
 };
 
-const cloneMetaData = (metaData) => {
-  // node js may not have structuredClone function, then fallback to deep clone via JSON
-  return typeof structuredClone === "function"
-    ? structuredClone(metaData)
-    : JSON.parse(JSON.stringify(metaData));
+const isFileValue = (value) => {
```

Comment:
I prefer to have functions underneath the place where they are used, so it reads more like a book. This is it feels like a constant cliffhanger of 'Why do we use this? Find out on the next line!' which makes it take more effort to understand the code.

Reply:
done